src/nett/brain/builder.py

```diff
@@ -322,10 +323,14 @@ def _validate_encoder(encoder: Any | str) -> BaseFeaturesExtractor:
         """
         # for when encoder is a string
         if isinstance(encoder, str):
-            if encoder not in encoder_dict.keys():
-                raise ValueError(f"If a string, should be one of: {encoder_dict.keys()}")
-            encoder = getattr(encoders, encoder_dict[encoder])
-
+            if encoder in encoders_dict.keys() and encoder != 'small':
+                encoder = getattr(encoders, encoders_dict[encoder])
+            elif encoder == 'small':
+                encoder = NatureCNN
+                encoder.__name__ = 'NatureCNN'
```

Comment:
Minor simplification / cleanup:
```suggestion
            if encoder == 'small':
                encoder = NatureCNN
                encoder.__name__ = 'NatureCNN'
            elif encoder in encoders_dict.keys():
                encoder = getattr(encoders, encoders_dict[encoder])
```